### PR TITLE
Add updated BBEs

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: examples/https-listener/https_listener.server.out [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # To start the service, navigate to the directory that contains the
 # `.bal` file and execute the `ballerina run` command.
```
Comment (Bhashinee) [inline examples/https-listener/https_listener.server.out]:
```suggestion
# `.bal` file and execute the `ballerina run` command along with the Ballerina home path as a config.
```

File: examples/mutual-ssl/mutual_ssl_service.out [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # To start the service, navigate to the directory that contains the
 # `.bal` file and execute the `ballerina run` command.
```
Comment (Bhashinee) [inline examples/mutual-ssl/mutual_ssl_service.out]:
```suggestion
# `.bal` file and execute the `ballerina run` command along with the Ballerina home path as a config.
```

File: examples/mutual-ssl/ssl_client.out [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # To start the service, navigate to the directory that contains the
 # `.bal` file and execute the `ballerina run` command.
```
Comment (Bhashinee) [inline examples/mutual-ssl/ssl_client.out]:
```suggestion
# `.bal` file and execute the `ballerina run` command along with the Ballerina home path as a config.
```

File: examples/basic-https-listener-client/basic_https_listener.out [hunk 4]
```diff
@@ -1,4 +1,4 @@
 # To start the service and run the main function, navigate to the directory that contains the
 # `.bal` file and use the `ballerina run` command.
```
Comment (Bhashinee) [inline examples/basic-https-listener-client/basic_https_listener.out]:
```suggestion
# `.bal` file and use the `ballerina run` command along with the Ballerina home path as a config.
```

File: examples/basic-https-listener-client/https_client.out [hunk 5]
```diff
@@ -1,4 +1,4 @@
 # To start the service and run the main function, navigate to the directory that contains the
 # `.bal` file and use the `ballerina run` command.
```
Comment (Bhashinee) [inline examples/basic-https-listener-client/https_client.out]:
```suggestion
# `.bal` file and use the `ballerina run` command along with the Ballerina home path as a config.
```

File: examples/header-based-routing/header_based_routing.server.out [hunk 6]
```diff
@@ -1,4 +1,5 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and use the `ballerina run` command.
-$ ballerina run header_based_routing.bal
+# `.bal` file and execute the `ballerina run` command along with the
+# Ballerina home path as a system property.
```
Comment (Bhashinee) [inline examples/header-based-routing/header_based_routing.server.out]: I think we shouldn't say it is a 'system property'.

Comment (chamil321) [inline examples/header-based-routing/header_based_routing.server.out]:
```suggestion
# Ballerina home path as a config.
```

Comment (chamil321) [reply examples/header-based-routing/header_based_routing.server.out]: Resolved

**Pull request merged**